### PR TITLE
Update portable job to Centos71

### PR DESCRIPTION
**Pull request by dseefeld**

### CODE REVIEW

File: .vsts.pipelines/builds/matrix.yml [hunk 1]
```diff
@@ -96,4 +96,4 @@ jobs:
         artifactName: Tarball debian9 Online
       - job: fedora30
       - job: fedora31
-      gatherPortableJob: centos71
+      gatherPortableJob: fedora30
```
Comment (omajid) [inline .vsts.pipelines/builds/matrix.yml:99]: Any particular reason for picking 30? It's been [EOL'ed for a while now](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/7UTUFY7WEL6RTFRXJB75XAFH44Y6RPUC/).

Comment (dseefeld) [reply .vsts.pipelines/builds/matrix.yml:99]: It's the one we're building Portable.  I'd rather not delay this PR any further, but this should be updated.  I filed an issue here: https://github.com/dotnet/source-build/issues/1780

**Comment by dagood** [comment]
What is this PR for?

**Comment by dseefeld** [comment]
> What is this PR for?

PrepareArtifacts failed because it was looking for CentOS71 Portable when we're actually building Fedora30 Portable.

**Review by dagood — APPROVED** [review]
I think the problem is using fedora30 for the portable build (more at https://github.com/dotnet/source-build/issues/1780) and I expect this will cause problems when trying to run `ilasm` and such in tarball builds when trying to uptake the previously-source-built bits, but approving in case it seems worthwhile to explore this direction first.

**Review by crummel — APPROVED** [review]

**Review by dagood — APPROVED** [review]

**Pull request merged**